src/main/flight/pid.c

```diff
@@ -445,12 +445,9 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     horizonFactorRatio = (100 - pidProfile->horizon_tilt_effect) * 0.01f;
     maxVelocity[FD_ROLL] = maxVelocity[FD_PITCH] = pidProfile->rateAccelLimit * 100 * dT;
     maxVelocity[FD_YAW] = pidProfile->yawRateAccelLimit * 100 * dT;
-    ITermWindupPointInv = 0.0f;
     if (pidProfile->itermWindupPointPercent < 100) {
         float ITermWindupPoint = (float)pidProfile->itermWindupPointPercent / 100.0f;
```

Comment:
Please add `const` and remove `(float)` cast

Reply:
Sorry to have missed that (again)